.github/workflows/continuous-integration-e2e.yml

```diff
@@ -81,12 +82,18 @@ jobs:
         run: npm run env:start
         env:
           COMPOSE_INTERACTIVE_NO_CLI: true
+          WP_VERSION: ${{ matrix.wp }}
 
-      - name: Run E2E tests
+      - name: Run E2E tests with percy
         run: npm run test:e2e:percy
         env:
           PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_E2E }}
           PUPPETEER_PRODUCT: ${{ matrix.browser }}
+        if: ${{ matrix.wp == 'latest' }}
+
+      - name: Run E2E tests
+        run: npm run test:e2e
+        if: ${{ matrix.wp != 'latest' }}
```

Comment:
Nit: in `continuous-integration-unit-php.yml` we use a separate `coverage` flag for a similar purpose:

https://github.com/google/web-stories-wp/blob/afb40799a5b2db28a55dda90dd7596706aac0268/.github/workflows/continuous-integration-unit-php.yml#L29-L33

https://github.com/google/web-stories-wp/blob/afb40799a5b2db28a55dda90dd7596706aac0268/.github/workflows/continuous-integration-unit-php.yml#L110-L120

Perhaps we could do the same here too with a `snapshot` flag for readability?

Reply:
I see you named it `coverage` as well now

Reply:
Sorry, I forgot that part when I was re-doing this. Should I change it to `snapshot`?